srcpkgs/zoom/template

```diff
@@ -16,6 +17,32 @@ noshlibprovides=yes
 noverifyrdeps=yes
 restricted=yes
 nopie=yes
+build_options="systemqt"
+desc_option_systemqt="Use system QT libraries"
+
+pre_install() {
```

Comment:
You can do that in do_install.

Reply:
It seemed fitting to do it before install, thus in pre_install. But I have no strong feelings about this, I can change it if you want.

Reply:
I think that build option should be added to the existing zoom package instead of creating a new one.